docs/pages/workflow/using-libraries.md

```diff
@@ -95,27 +109,37 @@ Now check the following:
+
+Many React Native libraries are not compatible with [Expo Go][expo-go]. For these libraries, you can create a [development build](/development/introduction):
 
-Many libraries you can use with Expo and React Native will not be compatible with Expo Go or `expo build:ios|android`. If you need any of them to build your app, you can create a [development build](../development/introduction.md) for your project using [EAS Build](../build/introduction.md) or [eject to the bare workflow](../workflow/customizing.md).
+<BoxLink title="Adding custom native code" description="Learn how to create a development build." href="/workflow/customizing" />
 
 > If you want some help determining library compatibility, [please create an issue on the React Native Directory repository](https://github.com/react-native-community/directory/issues/new/choose) and let us know. This will not just help you, it will help to ensure that other developers have an easy answer in the future!
 
 ### Installing a Third-Party Library
 
-> We recommend always using `expo install` instead of `npm install` or `yarn add` directly because it allows `expo-cli` to pick a compatible version of a library when possible and also warn you about known incompatibilities.
+> We recommend always using `npx expo install` instead of `npm install` or `yarn add` directly because it allows [Expo CLI][expo-cli] to pick a compatible version of a library when possible and also warn you about known incompatibilities.
 
-Once you have determined if the library is compatible, use `expo-cli` to install the package:
+Once you have determined if the library is compatible with React Native, use [Expo CLI][expo-cli] to install the package:
 
-<Terminal cmd={['$ npm install @react-navigation/native']} />
+<Terminal cmd={['$ npx expo install @react-navigation/native']} />
 
 Be sure to follow the project website or README for any additional configuration and usage instructions. You can get to the README quickly using this command:
 
 <Terminal cmd={['$ npx npm-home @react-navigation/native']} />
 
-If the module needs additional native configuration, you can do so using [config plugins](../guides/config-plugins.md).
+If the module needs additional native configuration, you can do so using [config plugins](/guides/config-plugins). Some packages require a config plugin but they don't have one yet, you can refer to the list of [out-of-tree config plugins](https://github.com/expo/config-plugins/).
+
+<ConfigReactNative>
+
+If your project does not support [Expo Prebuild](/workflow/prebuild) then you cannot use [config plugins](/guides/config-plugins). You can either [adopt Expo Prebuild](/guides/adopting-prebuild) or setup and configure each library manually by following any additional setup guides in the respective module's npm page.
```

Comment:
```suggestion
If your project does not support [Expo Prebuild](/workflow/prebuild) then you won't be able to use [config plugins](/guides/config-plugins). You can either [adopt Expo Prebuild](/guides/adopting-prebuild) or setup and configure each library manually by following any additional setup guides from the respective module's website or README.
```

i'm not crazy about this wording - "does not support Expo Prebuild" -- but let's try it, given that it's already scoped within a ConfigReactNative block